mkdocs/docs/configuration.md

```diff
@@ -62,7 +62,16 @@ You can also set the FileIO explicitly:
 | ---------- | -------------------------------- | ----------------------------------------------------------------------------------------------- |
 | py-io-impl | pyiceberg.io.fsspec.FsspecFileIO | Sets the FileIO explicitly to an implementation, and will fail explicitly if it can't be loaded |
 
-For the FileIO there are several configuration options available:
+General configuration:
```

Comment:
Shall we have a new section called `Tables` and make these configs under a sub-section `Write properties`. I think it may be better to separate the general table properties from configurations specific to FileIO and Catalogs, which are more vendor specific. 

Also, I think we can write a note in the new section, indicating that currently pyiceberg write only support certain table write properties. WDYT?